akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/StashBuffer.scala

```diff
@@ -8,22 +8,9 @@ import java.util.function.Consumer
 import java.util.function.{ Function => JFunction }
 
 import akka.actor.typed.Behavior
-import akka.actor.typed.internal.StashBufferImpl
 import akka.actor.typed.scaladsl
 import akka.annotation.DoNotInherit
 
-object StashBuffer {
```

Comment:
`StashBuffer` can/should be a trait now

Reply:
it is a trait?

Reply:
`abstract class StashBuffer` ?

Reply:
that is not important any more, was only a limitation of Scala 2.11

Reply:
Yah, just saying why it was made like this. I still think having it a pure interface when it is rather than an abstract class is better.

Reply:
Sorry I was being stupid, i was looking at the scaladsl which is a trait. I've switched the javadsl to a trait now